BUILDING.md

```diff
@@ -113,8 +113,8 @@ platforms. This is true regardless of entries in the table below.
 | Windows          | x86 (native)     | >= Windows 8.1/2012 R2          | Tier 1 (running) / Experimental (compiling) <sup>[6](#fn6)</sup> | |
 | Windows          | x64, x86         | Windows Server 2012 (not R2)    | Experimental |                                   |
 | Windows          | arm64            | >= Windows 10                   | Tier 2 (compiling) / Experimental (running) |    |
-| macOS            | x64              | >= 10.13                        | Tier 1       |                                   |
-| macOS            | arm64            | >= 11                           | Experimental |                                   |
+| macOS            | x64              | >= 10.14.4                      | Tier 1       |                                   |
```

Comment:
Actually do the binaries run on macOS 10.13? This table is the table of platforms we support running Node.js on but not a guarantee that you can build on this (building requirements are under the "Supported toolchains" section below).

The reason for asking is that usually we wouldn't change this table during a release (e.g. 16.x) and would normally consider raising versions to be semver-major.

Reply:
No idea as we dont test on 10.13 - the minimum version is set to target 10.13 but I dont think its ever been verified.

Reply:
This number really should match `MACOSX_DEPLOYMENT_TARGET` in common.gypi, and also see lines 175 and 176 below in this file.

`MACOSX_DEPLOYMENT_TARGET` has been a pretty solid tool historically for Node.js, it compiles the binaries to be compatible with the version we set there, even though we haven't always (maybe never) tested that far back. I think we should either match that value here (update it, or update this), or have some kind of note which makes it clear that we compile binaries to be supported on that version.

If you feel strongly enough that this column should be out of sync with `MACOSX_DEPLOYMENT_TARGET` then I'd suggest using the Notes column to describe what we do and that binaries _in theory_ should be compatible back to that version but _we recommend_ running it with the version you're putting here.

Reply:
I prefer that approach - Ill changed this back to 10.13 and add an note making it explicit that whilst we supporting our own binaries running on that level we dont necessarily support compiling at that level.